frb_codegen/src/method_utils.rs

```diff
@@ -124,33 +119,36 @@ impl FunctionName {
     }
 
     pub fn deserialize(s: &str) -> Self {
-        let actual_name: String;
-        let method_info = {
-            if MethodNamingUtil::is_static_method(s) {
-                actual_name = MethodNamingUtil::static_method_return_method_name(s);
-                MethodInfo::Static {
+        if MethodNamingUtil::is_static_method(s) {
+            FunctionName {
+                actual_name: MethodNamingUtil::static_method_return_method_name(s),
+                method_info: MethodInfo::Static {
                     struct_name: MethodNamingUtil::static_method_return_struct_name(s),
-                }
-            } else if MethodNamingUtil::is_non_static_method(s) {
-                actual_name = MethodNamingUtil::non_static_method_return_method_name(s);
-                MethodInfo::NonStatic {
+                },
+            }
+        } else if MethodNamingUtil::is_non_static_method(s) {
+            FunctionName {
+                actual_name: MethodNamingUtil::non_static_method_return_method_name(s),
+                method_info: MethodInfo::NonStatic {
                     struct_name: MethodNamingUtil::non_static_method_return_struct_name(s),
-                }
-            } else {
-                actual_name = s.to_string();
-                MethodInfo::Not
+                },
+            }
+        } else {
+            FunctionName {
+                actual_name: s.to_string(),
+                method_info: MethodInfo::Not,
             }
-        };
-        FunctionName {
-            actual_name,
-            method_info,
         }
     }
 
     pub fn method_name(&self) -> String {
         self.actual_name.clone()
     }
 
+    pub fn static_method_name(&self) -> String {
```

Comment:
nit: maybe this is `Option<String>`, and should `match &self.method_info { MethodInfo::Static=> return that thing; else => return none`